composer.json

```diff
@@ -52,15 +54,18 @@
         "xtreamwayz/pimple-container-interop": "^1.0 to use Pimple for dependency injection",
         "zendframework/zend-servicemanager": "^2.5 to use zend-servicemanager for dependency injection"
     },
+    "bin": [
+        "bin/expressive-pipeline-from-config"
+    ],
     "scripts": {
         "check": [
             "@cs-check",
             "@test"
         ],
         "upload-coverage": "coveralls -v",
-        "cs-check": "phpcs",
-        "cs-fix": "phpcbf",
-        "test": "phpunit",
+        "cs-check": "phpcs --colors",
+        "cs-fix": "phpcbf --colors",
```

Comment:
--colors for phpcs and phpcbf is not needed since the new coding standard will have them enabled via the ruleset.xml